stdlib/src/builtin/simd.mojo

```diff
@@ -1879,6 +1881,60 @@ struct SIMD[type: DType, size: Int](
 
         return bitcast[_integral_type_of[type](), size](self).cast[int_dtype]()
 
+    @staticmethod
+    fn from_bytes[
+        big_endian: Bool = False
+    ](bytes: InlineArray[Byte, Self.type_len]) raises -> Scalar[type]:
```

Comment:
Does this need to be `raises` anymore?

Reply:
Good catch! Removed here: https://github.com/msaelices/mojo/commit/40b9fc016ce7ebf7246a0e0c7db08aea00c445e4